src/app/modules/verification/__tests__/verification.service.spec.ts

```diff
@@ -295,6 +296,20 @@ describe('Verification service', () => {
     })
 
     it('should send OTP and update hashes when parameters are valid', async () => {
+      const mockForm = {
```

Comment:
this seems to be repeated a bunch of times, should we make it a constant just under describe block? (line 281?)

Reply:
thanks for highlighting, edited!